security/tor/src/opnsense/mvc/app/models/OPNsense/Tor/Relay.xml

```diff
@@ -8,8 +8,20 @@
     </enabled>
     <host type="TextField">
       <Required>N</Required>
-      <mask>/^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|[a-f0-9:]{2,})$/i</mask>
+      <mask>/^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}</mask>
```

Comment:
`$/` is missing - shouldn't that thow an exception on your machine? That's an invalid regular expression